admin-dev/themes/default/scss/modules/colors/_ambers.scss

```diff
@@ -1,25 +0,0 @@
-// Ambers 9
-// -------------------------
-
-$amber:               	#f90 !default;

```

Comment:
nice catch @jolelievre, I haven't seen anything like this but it is possible that somewhere, someone... :)

Reply:
Yes, I removed this part because I didn't see any occurrences of it across the back office. I will document the rebranding in the devdocs, where I can mention it.